dhis-2/dhis-api/src/main/java/org/hisp/dhis/schema/Property.java

```diff
@@ -837,6 +837,18 @@ public boolean is( PropertyType propertyType )
         return propertyType != null && propertyType.equals( this.propertyType );
     }
 
+    public boolean is( PropertyType... anyOf )
```

Comment:
isAnyOf could be a better name to understand exactly how this method works

Reply:
I had addressed this but Lars was faster than me with merging.